content/livesync/Setup

```diff
@@ -0,0 +1,190 @@
+---
+title: SDK setup
+meta_description: "Install, authenticate and instantiate the Ably Realtime Data Models SDK."
+product: LiveSync
+---
+
+Use these instructions to install, authenticate and instantiate the Ably Realtime Data Models SDK.
```

Comment:
Minor, but if listing these out, I would go install, instantiate and authenticate as thats the order those things happen

Although might be simpler to just say "Use these instructions to get started with" ?